acceptance/acceptance_test.go

```diff
@@ -0,0 +1,290 @@
+package acceptance_test
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"maps"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"slices"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/databricks/cli/internal/testutil"
+	"github.com/databricks/cli/libs/env"
+	"github.com/databricks/cli/libs/testdiff"
+	"github.com/stretchr/testify/require"
+)
+
+var KeepTmp = os.Getenv("KEEP_TMP") != ""
+
+func TestAccept(t *testing.T) {
+	execPath := BuildCLI(t)
+	t.Setenv("CLI", execPath)
+
+	server := StartServer(t)
+	AddHandlers(server)
+	t.Setenv("DATABRICKS_HOST", fmt.Sprintf("http://127.0.0.1:%d", server.Port))
+	t.Setenv("DATABRICKS_TOKEN", "dapi1234")
```

Comment:
Why do we need to set this token?

Reply:
To ensure it's not picked from environment. Also there is/was a bug where not having a token caused SDK to call itself in a loop. https://github.com/databricks/cli/pull/2076